src/backend/core/tests/api/test_admin_maildomains_create.py

```diff
@@ -0,0 +1,65 @@
+"""Tests for the MailDomain Admin API endpoints."""
+# pylint: disable=redefined-outer-name, unused-argument
+
+from django.urls import reverse
+
+import pytest
+from rest_framework import status
+
+from core import factories, models
+
+pytestmark = pytest.mark.django_db
+
+
+@pytest.fixture(name="domain_superuser_user")
+def fixture_domain_superuser_user():
+    """Create a user for domain superuser testing."""
+    return factories.UserFactory(is_superuser=True)
+
+
+@pytest.fixture(name="domain_admin_user")
+def fixture_domain_admin_user():
+    """Create a user for domain administration testing."""
+    return factories.UserFactory()
+
+
+@pytest.fixture(name="other_user")
+def fixture_other_user():
+    """Create another user without admin privileges."""
+    return factories.UserFactory()
+
+
+class TestAdminMailDomainsCreate:
+    """Tests for the MailDomain Admin API create endpoint."""
+
+    CREATE_DOMAIN_URL = reverse("admin-maildomains-list")
```

Comment:
Didn't find a better candidate using `python manage.py show-urls`, as I am using the default creation method. Is that ok to use ?

Reply:
it's fine, the reverses are a bit of a mess indeed. internal details, we can change later.